tensorflow-examples/src/main/java/org/tensorflow/model/examples/mnist/data/ImageBatchIterator.java

```diff
@@ -22,7 +22,7 @@
 import org.tensorflow.tools.ndarray.ByteNdArray;
 import org.tensorflow.tools.ndarray.index.Index;
 
-class ImageBatchIterator implements Iterator<ImageBatch> {
+public class ImageBatchIterator implements Iterator<ImageBatch> {
```

Comment:
These should probably have a little bit of documentation if they are public.

Reply:
Agree